fastlane/Fastfile

```diff
@@ -428,6 +429,70 @@ platform :android do
     )
   end
 
+  desc 'Builds a prototype build and uploads it to S3'
+  lane :build_and_upload_prototype_build
+    UI.user_error!("'BUILDKITE_ARTIFACTS_S3_BUCKET' must be defined as an environment variable.") unless ENV['BUILDKITE_ARTIFACTS_S3_BUCKET']
+
+    gradle(
+        task: "assembleRelease",
+        properties: {
+        "skipSentryProguardMappingUpload" => true
+      }
+    )
+
+    prototype_build_comments = lane_context[SharedValues::GRADLE_ALL_APK_OUTPUT_PATHS].map { |apk_path|
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-android/blob/14fbc28fd1fb46d68ff9627a26aa8717dc4f1c77/fastlane/Fastfile#L443"></span>Style/BlockDelimiters: Avoid using <code>{...}</code> for multi-line blocks.</td>
    </tr>
  </tbody>
</table>